pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -39,7 +39,7 @@
     <!-- Permit the following implementation-only assemblies -->
     <ValidatePackageSuppression Condition="'$(PackageTargetRuntime)' != ''" Include="PermitInbox">
       <Value>
-        Microsoft.VisualBasic;
+        Microsoft.VisualBasic.Core;
```

Comment:
I don't believe this is required since this is not implementation only.  We can delete the line as was done for netcoreapp.